### ISSUE TITLE
Show, don’t tell! How I landed a job at a tech startup without a CV

### ISSUE THREAD

**Issue by sgrove** [sgrove/rfcrf-tanmaioneblog#18]
_Every six months, we run an **online course that teaches beginners how to build applications (called [IMAD](http://bit.ly/arpit-cv))**. Each session gets over 100,000 students, and while we strive to be responsive to all of their requests and queries, it just wouldn’t be possible to manage an audience of this size without some **student volunteers** who display fantastic initiative._

_From day one, “Shark” stood out. We were so impressed with his incredible initiative and relentless drive that we created an internship just for him. When he displayed the same tenacity during his time at Hasura, [Tanmai](https://www.twitter.com/tanmaigo) (Co-founder) offered him a job the day his internship ended._

_Here’s how he convinced us to hire him without a formal interview process in his own words._

![](https://github.com/sgrove/rfcrf-tanmaioneblog/blob/master/img/704/1*IvEv8hrbc2MpromlPmLnXQ.jpeg?raw=true "You can see where “Shark” comes from")

**TL;DR:**

-   **I volunteered for the MOOC that Hasura was running and helped out**

-   **This lead to them offering me a position as a T.A and then as an intern**

-   **My internship came with a lot of responsibilities that I worked hard at.**

-   **They were so happy with my performance that they ended up offering me a job.**

## Introduction

Hasura is a cloud infrastructure company based out of Bangalore, focussed on making product development easier. It’s tech-focussed and it’s where I got my first job. I landed my position with Hasura **without interviewing with them once. I didn’t even show them my CV.**

How did I do it?

Have you heard that famous Paulo Coelho quote? You know, this one…

![](https://github.com/sgrove/rfcrf-tanmaioneblog/blob/master/img/704/1*YKasarGUoiaxsV0UZb98Jw.jpeg?raw=true "When you wish upon a star")

It wasn’t like that at all. It wasn’t:

-   The Secret (the book)
-   I didn’t spam them till they gave up and offered me a job in exchange for stopping
-   There was no serendipitous meeting with the founders in an elevator.

I didn’t need to interview or show a CV because **I had spent nearly a year working with them before landing a full-time role**. They knew exactly what they were getting when they hired me. Before graduating from college, I’d worked with Hasura, first as a volunteer, then as a T.A and finally as an intern.

That’s the story in a nutshell. Here’s the long version:

## Discovering IMAD

I first came across **IMAD** when my friend insisted that I take the course. I’d never heard of [NPTEL](http://nptel.ac.in/) before, let alone IMAD (which I thought was a strange name for an educational program). I was even more apprehensive when I came across NPTEL’s dated site. It looked like yet another dull online course.

I couldn’t have been more wrong.

I’ve always been good at computers, programming came quite naturally to me. But IMAD was a different beast. Once I got started with the course it was impossible to stop! The videos were simple and clear. It felt like they were talking to me rather than just teaching. **Tanmai**’**s** ability to explain complex topics in simple terms was something anyone would appreciate.

While I’m good at programming, I actually dislike writing code. This wasn’t an issue with IMAD though. The assignments were a lot of fun (especially when the program works the first time :D). But the course being super interesting wasn’t **the best part.** When I took IMAD, I stumbled upon a Passion® that I didn’t know I had.

## The forum

Thanks to already being familiar with coding, the tutorial videos being great and the interesting assignments — I ended up finished the course well ahead of schedule.

IMAD is aimed at people with no tech background. But from day one it stresses getting your hands dirty with code. You constantly learn-by-doing. This meant that people who were new to coding would sometimes be stumped and frustrated. In my opinion, the course does a great job of explaining the concepts involved. But at the end of the day, there’s no way around the fact that **for first timers, coding will be challenging**. It’s just part of the game. If you read the testimonials from other IMAD students, you’ll notice that every single one of them mentions how great the forum was. The forum became the hub around which people get doubts cleared and problems sorted out. Students really lean on each other for support and find mentors amongst peers. It’s a really great learning experience. It’s also a fantastic way to reinforce whatever you’ve learned. If you can’t help someone out with a problem they have, you probably need to revise that section.

In my spare time, I started hanging out on the course forum, interacting with other students and helping them out. Slowly, I started spending more and more time here since I saw a lot of people getting stuck and frustrated. What most students needed was a bit of patience and some encouragement. Most of my time was spent reading their code along with them. We would go through each line and I would either explain what went wrong or I would nudge them to figure it out themselves.

Encouraging others not to quit was also something I spent a lot of time on. Like I said earlier, coding will be frustrating at times. A lot of people who were new to coding would eventually get so frustrated that they’d want to quit. When I picked up on a thread where someone was saying that they were ready to drop out, I would immediately step in and have a little chat with them. I’d try to figure out what went wrong: did they think coding was beyond them? Were they not able to figure out how to solve a problem? I’d try to get to the root of the issue and try to talk them out of it. Sometimes I’d even send them cheesy motivational messages encouraging them not to quit.

## Discovering the Passion®

It turned out that I preferred helping people out more than writing code. Eventually, I stopped coding completely and spent all my time helping others on the forum. It slowly dawned on me that helping someone realise it’s not that difficult to write programs was more rewarding for me than actually writing code. **This was new!** It was great to observe people overcome mistakes, mental blocks and setbacks to continue on with the course.

At this point, I was still a volunteer but that didn’t matter. **I was regularly up till 1 a.m helping out others and clearing their doubts**. Eventually, the IMAD team asked me to work as a TA on the next session of IMAD. As a TA my scope of activities didn’t change much. I was already putting in so much time on the forums that a switch to an official position barely changed things at all.

## **The Internship**

One day I got a mail from [**Ashish**](https://www.facebook.com/Ashish.Kashyap.IITM) (head of IMAD) asking me if I would be interested in interning at Hasura. Since I’d never worked outside my hometown I jumped at the idea.

I ended up working on both IMAD as well as the new **[Hasura Product Development Fellowship](https://goo.gl/E3Nrwf)** (HPDF). HPDF is a post-IMAD internship where Hasura engineers mentor the top 100 IMAD students and help them turn their ideas into apps.

Here’s a snapshot of my activities during my internship:

**IMAD specific activities**

-   Worked with the IMAD team to grow the course.
-   Managed the forum — set and enforced rules of conduct, helped people out and generally made sure that students had the best experience possible.
-   Worked with Ashish and Tanmai to modify course content.
-   Got Shahidh to fix any issues that popped up on the console backend.
-   Answered questions about the course.

**HPDF specific activities**

-   Selected candidates and on-boarded them.
-   Helped set the curriculum.
-   Organized guest lectures.
-   Liaised with the Fellows.
-   Judged the final apps.

As you can tell, I had a lot on my plate. The internship was filled with long days but I didn’t mind. **Working at Hasura was unlike anything I’d experienced before**: You are treated as an equal and are given the freedom and responsibility to solve problems yourself. When you give your opinions, they are taken seriously. This is incredible but also really scary because I was doing a lot of these activities for the first time.

On my last day, Tanmai invited me into a room and offered me a job. In all honesty, I really didn’t expect this at all. While I was done with my finals, job hunting was still the furthest thing on my mind. That being said, when I was offered the job, I jumped at it.

## What can you learn from all of this?

While the title might be a little click-baity, it was the truth. I really did not interview with them nor did I need a CV. But like I stressed, this was because they got multiple chances to see how I worked. My performance was my CV & my interview.

Big companies have policies and standards that govern hiring. They have very clear requirements and job descriptions. They know exactly what they want and will rarely ever deviate from these norms. This makes their hiring practices rigid.

Startups / young companies lack the formal processes of larger enterprises. They really need people with a lot of initiative, who’re able to operate in ambiguous situations and exhibit a readiness to learn new skills. Don’t take the fact that they haven’t advertised for a position as final. If you can show them you’re someone who can come up with a logical plan, execute it well, learn fast and that you’re dependable, chances are that they will jump at the chance to hire you.

What’s key is to figure out a way to get your foot in the door. Volunteering is a great way to do this because it lets them see how you get things done. I did this by helping out on the online course forum, but there are many other routes you can take. If you identify a company you want to work for, look out for ways to help them out, take it step by step, and build a relationship. If it doesn’t result in a job, there’s always a chance it could result in other positive outcomes.

You never know.

**P.S. Also, I loved the awesome, heavy package the IMAD team sent me!**

![](https://github.com/sgrove/rfcrf-tanmaioneblog/blob/master/img/704/1*dqWFOiUtFkL9sasl09mGlg.jpeg?raw=true "An IMAD goodie bag")

---

_Aimed at anyone, with or without a technical background, [**Introduction to Modern Application Development (IMAD)**](https://www.imad.tech/?utm_source=Medium&utm_medium=Blog%20Post&utm_campaign=Imad%20Stories&utm_content=Arpit) is an online course from IIT + [Hasura](https://goo.gl/8CWYJh) that teaches application development fundamentals. If you’ve ever wanted to build an web/mobile application yourself or just understand how they work, [**you should check it out**](https://www.imad.tech/?utm_source=Medium&utm_medium=Blog%20Post&utm_campaign=Imad%20Stories&utm_content=Arpit)._

```backmatter
{"source":"medium","postId":"709596da3bc1","publishedDate":1512989691081,"url":"https://medium.com/@HasuraHQ/show-dont-tell-how-i-landed-a-job-at-a-tech-startup-without-a-cv-709596da3bc1"}
```
